src/symex/path_search.cpp

```diff
@@ -36,10 +36,16 @@ path_searcht::resultt path_searcht::operator()(
   
   locs.build(goto_functions);
 
-  // this is the container for the history-forest  
+  // Perform a non-comprehensive check on JSON file.
+  if(taint_engine->enabled &&
```

Comment:
Indeed.  Is there a preferred syntax within CProver for checking for null-ness?

(C++ supports a few methods)